Tchap/Modules/Contacts/DataSources/ContactsDataSource.m

```diff
@@ -1216,21 +1216,11 @@ - (BOOL)tableView:(UITableView *)tableView canEditRowAtIndexPath:(NSIndexPath *)
 
 #pragma mark -
 
--(BOOL)isInviteButtonIndexPath:(NSIndexPath*)indexPath
-{
-    return (indexPath.section == inviteToTchapButtonSection);
-}
-
 -(BOOL)isInviteByLinkButtonIndexPath:(NSIndexPath*)indexPath
 {
     return (indexPath.section == inviteByLinkButtonSection);
 }
 
--(BOOL)isAddEmailButtonIndexPath:(NSIndexPath*)indexPath
```

Comment:
@phlniji we have to keep this code to be able to invite by email in an existing room